source/includes/administration/_email_settings.md

```diff
@@ -0,0 +1,171 @@
+## Email Settings
+
+<!-------------------- GET EMAIL SETTINGS -------------------->
+
+### Retrieve email settings
+
+`GET /email_settings/:id`
+
+```shell
+# Retrieve knowledge base
+curl "https://cloudmc_endpoint/rest/brandings/dc30f9a6-1edc-11eb-92b6-0242ac120003" \
```

Comment:
/rest/email_settings/:id

Reply:
that is ok... it is an example. The structure is as line 7